source/utils/Helper.h

```diff
@@ -30,6 +30,10 @@
 #include <vector>
 #include <cstdint>
 
+#ifdef _MSC_VER
+    #define snprintf _snprintf
```

Comment:
Yes, please, use the C++ example of CEGUI for instance. I've got nothing against C, but here it's not worth the exception.